src/main/java/dk/cngroup/lentils/entity/formEntity/HintFormObject.java

```diff
@@ -0,0 +1,63 @@
+package dk.cngroup.lentils.entity.formEntity;
+
+import dk.cngroup.lentils.entity.Cypher;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotEmpty;
+import java.io.Serializable;
+
+public class HintFormObject implements Serializable {
```

Comment:
Tááák, po posledních změnách v masteru by se teď tohle mělo jmenovat HintFormDTO a mělo by to být v balíčku dk.cngroup.lentils.dto  :)